### ISSUE TITLE
Mojit-level addons are not recognized by Mojito.

### ISSUE THREAD

**Issue by zhouyaoji** [YahooArchive/mojito#875]
Based on the documentation, you can put addons at the application or mojit level.
See the following: http://developer.yahoo.com/cocktails/mojito/docs/topics/mojito_extensions.html#addons

Addons at the application level work as expected, but when addons are at the mojit level, e.g., ({app_name}/mojits/{mojit_name}/addons/ac/{add_on}, Mojito does not find the addon.

The following example has both application and mojit-level addons. The application throws an error because the addon is not found: https://github.com/zhouyaoji/Mojito-Apps/tree/master/addon_ex/

**Comment by drewfish** [comment]
I think that example might be broken.  Compare the YUI module name of the AC addon versus the one that the controller is depending on:
https://github.com/zhouyaoji/Mojito-Apps/blob/master/addon_ex/mojits/myMojit/addons/ac/mojitlevel.common.js
https://github.com/zhouyaoji/Mojito-Apps/blob/master/addon_ex/mojits/myMojit/controller.server.js
The controller is depending on `addon-ac-mojitlevel` yet that addon has a YUI module name of `addon-ac-cheese`.

**Comment by zhouyaoji** [comment]
Thanks, Drew, you're right. I was having an issue using a mojit-level addon when trying to convert Ric Allinson's hybrid app to use Mojito v0.5, so I created my own example to recreate the problem. I recreated the problem, but the real problem was the registered YUI module name of the addon Once I change the registered YUI module name, the mojit-level addon works correctly. I'll have to take another look at the hybrid app. I'll close this (non)issue.
